include/libunwind-x86.h

```diff
@@ -158,6 +158,7 @@ x86_regnum_t;
 typedef struct unw_tdep_save_loc
   {
     /* Additional target-dependent info on a save location.  */
+    char unused;
```

Comment:
Maybe better for consistency between arch variants to use the same `UNW_EMPTY_STRUCT` idiom as was used in libunwind-aarch64.h and libunwind-arm.h ?

Reply:
Done 👍